steps/hearing/arrangements/signLanguages.js

```diff
@@ -0,0 +1,15 @@
+module.exports = [
+  'American Sign Language (ASL)',
+  'British Sign Language (BSL)',
+  'Deaf Blind ‑ Hands on Sign Language',
+  'International Sign Language',
+  'Lip Speaker Sign Language',
+  'Makaton Sign Language',
+  'Manual Alphabet',
+  'Note taker',
+  'Relay',
+  'SSE',
+  'Visual Frame',
+  'Sign Language (Others)',
+  'Sign Language Palantypist/Speech to TX'
+];
```

Comment:
This needs a new line and also a basic unit test. Simply just to check it exports an array